dev/tests/integration/testsuite/Magento/ConfigurableProduct/_files/product_configurable_zero_qty_first_child.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
Let's include `configurable` fixture here and set zero quantity for the child by `sku`.